indigo_app/forms.py

```diff
@@ -679,7 +679,7 @@ def filter_queryset(self, queryset, exclude=None):
         if self.cleaned_data.get('sortby'):
             queryset = queryset.order_by(self.cleaned_data.get('sortby'))
 
-        return queryset
+        return queryset.distinct()
```

Comment:
Try `.distinct("pk")` so that it only does the distnict on the primary key, and not on everything. I haven't tested.

Reply:
it didn't like that 😅

Reply:
never mind